concourse/scripts/pipelines-bosh-cloudfoundry.sh

```diff
@@ -38,6 +38,7 @@ prepare_environment() {
   cf_release_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.cf.version "${cf_manifest_dir}/000-base-cf-deployment.yml")
   cf_graphite_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.graphite.version "${cf_manifest_dir}/055-graphite.yml")
   cf_grafana_version=$("${SCRIPT_DIR}"/val_from_yaml.rb releases.grafana.version "${cf_manifest_dir}/055-graphite.yml")
+  gpg_ids="$(awk '{printf "\\n        - %s ", $1}; END {printf "\\n"}' < "${SCRIPT_DIR}"/../../.gpg-id)"
```

Comment:
Rather than creating a string with new lines and the right amount of spaces, I suggest you use an inline list `[gpg1,gpg2]`. This way you don't need to "know" the exact number of spaces in this script:

For instance:

```diff
--- a/concourse/scripts/pipelines-deployer.sh
+++ b/concourse/scripts/pipelines-deployer.sh
@@ -32,9 +32,9 @@ generate_vars_file > /dev/null # Check for missing vars
 
 generate_manifest_file() {
   if ${COMMIT_VERIFICATION} ; then
-    gpg_ids="$(awk '{printf "\\n        - %s ", $1}; END {printf "\\n"}' < "${SCRIPT_DIR}"/../../.gpg-id)"
+    gpg_ids="[$(cat "${SCRIPT_DIR}/../../.gpg-id" | xargs | tr ' ' ',')]"
   else
-    gpg_ids="null"
+    gpg_ids="[]"
   fi
 
   # This exists because concourse does not support multiline value interpolation by design

```